test/vmtests/vmtests/utils/libvirt_utils.py

```diff
@@ -158,18 +190,32 @@ def create_libvirt_domain_xml(libvirt_conn: libvirt.virConnect, vm_spec: VmSpec)
 
     devices = ET.SubElement(domain, "devices")
 
+    if host_arch == "aarch64":
+        emulator = ET.SubElement(devices, "emulator")
+        emulator.text = "/usr/bin/qemu-system-aarch64"
+
+        controller_scsi = ET.SubElement(devices, "controller")
+        controller_scsi.attrib["type"] = "scsi"
+        controller_scsi.attrib["index"] = "0"
+        controller_scsi.attrib["model"] = "virtio-scsi"
+
     serial = ET.SubElement(devices, "serial")
-    serial.attrib["type"] = "pty"
+    serial.attrib["type"] = "file"
+    serial_source = ET.SubElement(serial, "source")
+    serial_source.attrib["path"] = log_file
```

Comment:
In LISA, I wrote a class that manually listens to and logs the serial console using the libvirt API: [console_logger.py](https://github.com/microsoft/lisa/blob/main/lisa/sut_orchestrator/libvirt/console_logger.py). If we ever want live-logging, we might want to use that instead. But if this works, it is probably fine for now.

Reply:
Yep, this captures the logs, Chris. It's not ideal because I can't print them live like the class you though.
I'll leave as-is for the sake of time.